### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by davec504**
Update repo url

### CODE REVIEW

**Review by ACoolmanTelicent — APPROVED** [review]
lgtm

File: package.json [hunk 1]
```diff
@@ -13,14 +13,14 @@
   "private": true,
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/Telicent-io/js-ontology-lib.git"
+    "url": "git+https://github.com/Telicent-oss/rdf-libraries.git"
```
[the remark this answers is not in the document]
Comment (davec504) [reply package.json:16]: Good point, I have actioned those

[the remark this answers is not in the document]
Comment (tatelicent) [reply package.json:16]: Need to update the names in the others as well I think

File: packages/OntologyService/package.json [hunk 2]
```diff
@@ -26,10 +26,10 @@
...
     "type": "git",
-    "url": "git+https://github.com/Telicent-io/js-ontology-lib.git"
+    "url": "git+https://github.com/telicent-oss/rdf-libraries.git"
   },
   "publishConfig": {
-    "registry": "https://npm.pkg.github.com/"
+    "registry": "https://www.npmjs.com/"
   },
   "dependencies": {
     "@telicent-io/rdfservice": "*",
```
Comment (tatelicent) [inline packages/OntologyService/package.json:35]: Probably need to update this too?

**Comment by davec504** [comment]
> Have you checked the lock file too?

I didn't see anything and didn't expect to because when you pull this down locally and run using nx it essentially does a yarn link for local package dependencies

**Comment by tatelicent** [comment]
> > Have you checked the lock file too?
> 
> I didn't see anything and didn't expect to because when you pull this down locally and run using nx it essentially does a yarn link for local package dependencies

Might wanna check the ds?

**Review by tatelicent — APPROVED** [review]
LGTM

**Review by ACoolmanTelicent — APPROVED** [review]
lgtm x2

**Pull request merged**